src/index.ts

```diff
@@ -2,9 +2,24 @@ export { supportedDialects, format, formatDialect } from './sqlFormatter.js';
 export { expandPhrases } from './expandPhrases.js';
 export { ConfigError } from './validateConfig.js';
 
-// Intentionally use "export *" syntax here to make sure when adding a new SQL dialect
-// we wouldn't forget to expose it in our public API.
-export * from './allDialects.js';
+// When adding a new dialect, be sure to add it to the list of exports below.
```

Comment:
Removing the `export *` was necessary because esbuild transformed it into code that wasn't statically analyzable for tree shaking. If desired, I can look into adding a test that asserts that this file has all the necessary language imports?